grammars/asciidoc.cson

```diff
@@ -64,17 +80,123 @@
     'begin': '(^/{4,}$)'
     'end': '\\1'
     'name': 'comment.block.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches example block
+  #
+  # Examples
+  #
+  #   ====
+  #   A multi-line example.
+  #
+  #   Notice it's a delimited block.
+  #   ====
+  {
+    'begin': '^(={4,})+?$'
+    'end': '\\1'
+    'name': 'markup.block.example.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches sidebar block
+  #
+  # Examples
+  #
+  # ***********************************
+  # As an aside, I love Asciidoc
+  # ***********************************
+  {
+    'begin': '(^\\*{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.sidebar.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches literal block
+  #
+  # Examples
+  #
+  # ...................................
+  # Daleks EXTERMINATE in monospace!
+  # ...................................
+  {
+    'begin': '(^\\.{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.literal.asciidoc'
+    'patterns': [
+      { 'include': [ '#inline', '#codeBlocks'] }
```

Comment:
According to the [manual](http://asciidoctor.org/docs/user-manual/#built-in-blocks-summary) substitution rules do not apply to literal blocks. I think we don't need to include inline and code block patterns here.